### DISCUSSION TITLE
Welcome to Kryptokrona Kotlin SDK Discussions!

### DISCUSSION THREAD

**Discussion by mjovanc** [kryptokrona/kryptokrona-kotlin-sdk#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with our Kotlin SDK.

**Comment by dev-divyansh** [comment]
Hello devs ,
I am Divyansh , currently learning Android Development ;
I want to contribute to this project but I don't have deep understanding of Android ;
Let me know how i can get started

**Reply by mjovanc** [reply]
Welcome! :smiley: To be clear, this is not specifically for Android. But targets Android, server-side or anything really using the JVM or Kotlin Native. The best thing at the moment is to browse through issues and just check the source code for improvements in form of more clearer and concise syntax.  

I will try take more time to make the issues more descriptive and clearer as well as add new ones that I can think of that is suitable for beginners of this project. Keep an eye at the label on the issues with "good first issue".
